hilo.rb

```diff
@@ -1,2 +1,22 @@
-number = rand(1..100)
-puts number 
\ No newline at end of file
+random_number = rand(1..100)
+puts "Enter a number between 1 and 100. press enter to submit your guess"
+input = gets.chomp
+guess = input.to_i
+response = ''
+count = 0
+while count < 5
+    count = count + 1
+    response = ''
+    if guess > 100
+        response = "Your guess is too high. Try again"
+    elsif guess < 1
+        response = "Your guess is either too low or is not a number. Try again"
+    else
+        response = (guess == random_number) ? break : "Incorrect! Try again"
```

Comment:
I think you can get away with just a break here.

Reply:
ok - but then where would the user get told that their answer is incorrect?

Reply:
I might be wrong, but based on the rules we have in the Hi-Lo brief I think we don't need to inform the player that their answer is incorrect.  Only that we inform the player its it high, low or correct.

Reply:
oh ok - so shall I remove that?